src/init.cpp

```diff
@@ -157,7 +156,8 @@ NODISCARD static bool CreatePidFile()
 static std::unique_ptr<ECCVerifyHandle> globalVerifyHandle;
 
 static boost::thread_group threadGroup;
-static CScheduler scheduler;
+
+CScheduler g_scheduler;
```

Comment:
Is it a lot of work to put this into the node context in this PR? Then you can avoid having the diff of renaming g_scheduler to m_scheduler or whatever...

Reply:
I also wouldn't even bother renaming it to g_scheduler or m_scheduler. It just makes your diff bigger, and someone else can fix it later.

Reply:
Ah I guess if they are a part of nodecontext then you'll need to hit these lines anyways.

Reply:
ya I'm planning to update to pass scheduler through node context instead of making it a global.

Reply:
I'd also put a comment that this is TODO